library/camel-kamelets-catalog/src/main/resources/kamelets/regex-router-action.kamelet.yaml

```diff
@@ -0,0 +1,42 @@
+    camel.apache.org/provider: "Red Hat"
+    camel.apache.org/kamelet.group: "Actions"
+spec:
+  definition:
+    title: "Regex Router Action"
+    description: "Update the destination using the configured regular expression and replacement string"
+    required:
+      - regex
+      - replacement
+    properties:
+      regex:
+        title: Regex
+        description: Regular Expression for destination
+        type: string
+      replacement:
+        title: Replacement
+        description: Replacement when matching
+        type: string
+    type: object
+  dependencies:
+  - "github:openshift-integration.kamelet-catalog:camel-kamelets-utils:main-SNAPSHOT"
```

Comment:
I think we are still on master here

Reply:
The pom version used is main-SNAPSHOT

Reply:
Yeah, that needs also to be changed.. let's merge this and change it everywhere